index.html

```diff
@@ -0,0 +1,136 @@
+    rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <header class="container">
+  <!-- start of nav section-->
+  <nav class="nav-wrapper">
+    <!--nav left side-->
+    <div class="nav-left">
+      <a href="#">My Logo</a>
+    </div>
+    <!-- nav right side-->
+    <div class="nav-right">
+
+        <button class="hamburger"><i class="fa fa-bars"></i></button>
+
+    </div>
+  </nav>
+  </header>
+  <!--end of nav -->
+
+  <!--start of  section-->
+    <section class="section-wrapper">
+    <div class="section-header">
+      <h1>I'm Abdullah<br> Glad to see you!</h1>
+    </div>
+    <div class="content-of-section">
+      <p>I'm a software developer!<br> I can help you build a product feature or a website. 
+        look through some of my work and experience if you like what you see and 
+        have a product you needed coded, don't hestiate to contact me.
+      </p>
+    </div>
+
+    <div class="social-icons">
+      <div class="icons-header">
+        <h2>LET'S CONNECT</h2>
+      </div>
+      <ul class="icons-list">
+        <li><a href="#" class="icon-link"><i class="fa fa-twitter"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-github"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-linkedin"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-medium"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-angellist"></i></a></li>
+      </ul>
+    </div>
+
+    </section> 
+      <!--work section-->
+      <section class="container-work">
+        <div class="card-info">
+        <div class="card-image">
+          <img src="images/Portfolio.png" alt="">
+        </div>
+        <div class="card-header">
+          <h3>Multi-post <br> stories</h3>
+        </div>
+        <ul class="card-details">
+          <li class="card-first-element">CANOPY</li>
+          <li class="circle-icon"><i class="fa fa-circle"></i></li>
+          <li class="card-second-element">Frontend Dev</li>
+          <li class="circle-icon"><i class="fa fa-circle"></i></li>
+          <li class="card-third-element">2022</li> 
+          </ul>
+        <div class="card-content">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <ul class="btn-button">
+          <li class="btn-languages"><a href="#" class="btn-link">HTML</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">CSS</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">Javascript</a></li>
+        </ul>
+        <button class="btn-card">See project</button>
```

Comment:
Kindly this section should display the image of each projects. and the project should have the see project button according to the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)